README.md

```diff
@@ -15,12 +15,16 @@ A boilerplate for building static sites with [React][] and [React Router][]
 
 Now you're all set to deploy to your favorite hosting solution :beers:
 
+To check your production build
+`npm install pushstate-server -g`
```

Comment:
pushstate-server is a static file server that mentioned in creact-react-app. It solves desire to see production code with couple lines in the terminal :) I just wanted to see production build but I didn't have nginx on my osx machine. This dependency doesn't force me to download brew then nginx and finally get an exception with nginx config what I actually got.